Crash report 2.7.1 - NPE in SetupBlog

```
java.lang.RuntimeException: An error occured while executing doInBackground()
at android.os.AsyncTask$3.done(AsyncTask.java:300)
at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
at java.util.concurrent.FutureTask.run(FutureTask.java:242)
at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
at java.lang.Thread.run(Thread.java:841)
Caused by: java.lang.NullPointerException
at org.wordpress.android.ui.accounts.SetupBlog.isHTTPAuthErrorMessage(SetupBlog.java:190)
at org.wordpress.android.ui.accounts.SetupBlog.getmXmlrpcByUserEnteredPath(SetupBlog.java:214)
at org.wordpress.android.ui.accounts.SetupBlog.getSelfHostedXmlrpcUrl(SetupBlog.java:284)
at org.wordpress.android.ui.accounts.SetupBlog.getBlogList(SetupBlog.java:104)
at org.wordpress.android.ui.accounts.WelcomeFragmentSignIn$SetupBlogTask.doInBackground(WelcomeFragmentSignIn.java:431)
at org.wordpress.android.ui.accounts.WelcomeFragmentSignIn$SetupBlogTask.doInBackground(WelcomeFragmentSignIn.java:401)
at android.os.AsyncTask$2.call(AsyncTask.java:288)
at java.util.concurrent.FutureTask.run(FutureTask.java:237)
... 4 more
```

--- a/src/org/wordpress/android/ui/accounts/SetupBlog.java
+++ b/src/org/wordpress/android/ui/accounts/SetupBlog.java
@@ -187,7 +187,7 @@ public class SetupBlog {
     }
 
     private boolean isHTTPAuthErrorMessage(Exception e) {
-        if (e != null && e.getMessage().contains("401")) {
+        if (e != null && e.getMessage() != null && e.getMessage().contains("401")) {
             mHttpAuthRequired = true;
             return mHttpAuthRequired;
         }

